esx_service/vmdk_ops_test.py

```diff
@@ -238,5 +238,6 @@ def test_failure(self):
         if not paths:
             logging.debug("Directory clean up - removing  %s", path)
             os.removedirs(path)
-    #suite = unittest.TestLoader().loadTestsFromTestCase(TestStringMethods)
-    #unittest.TextTestRunner(verbosity=2).run(suite)
+
+        # If the unittest failed, re-raise the error
+        raise
```

Comment:
All tests in this module and all other python tests in test-esx will continue to run. Any tests not under the `test-esx` target in the Makefile will not however. This is just fixing a bug that I assume was recently introduced here, in which a failing test didn't return a failing (non-zero) exit code from the process. All other python test files end with `unittest.main()` which will run through all tests even if one fails, but return a non-zero exit code if there is actually a failure after they all run.